pom.xml

```diff
@@ -185,5 +185,10 @@
       <artifactId>JRI</artifactId>
       <version>0.9-7</version>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
```

Comment:
it supports more sophisticated junit assertions like greaterThan, it leads to much more readable test failures than the failure of some opaque boolean flag.